src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.WinRT.cs

```diff
@@ -39,6 +39,11 @@ private WebSocketHandle(WinRTWebSocket webSocket)
         
         public async Task ConnectAsyncCore(Uri uri, CancellationToken cancellationToken, ClientWebSocketOptions options)
         {
+            if (options.RemoteCertificateValidationCallback != null)
+            {
+                throw new PlatformNotSupportedException();
```

Comment:
We should add a string message to this exception saying something like "ClientWebSocketOptions.RemoteCertificateValidationCallback not supported".

It isn't going to be obvious to a developer that getting this PNSE exception in ConnectAsync codepath is related to them previously setting this callback.

Reply:
Ok